core/commands/files.go

```diff
@@ -757,6 +759,14 @@ stat' on the file or any of its ancestors.
 			return
 		}
 
+		if mkParents {
+			err := ensureContainingDirectoryExists(nd.FilesRoot, path, flush, prefix)
```

Comment:
No point in flushing here. We'll flush when we actually create the file.

Reply:
Ok, I've changed it to not pass a `Flush` option to `mfs.Mkdir`